src/pages/Notes/NewNote.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from "react";
+import ColorOptions from "../../components/ColorOptions";
+
+function NewNote({ addNote }) {
+  const [takingNote, setTakingNote] = useState(false);
+  const [openMenu, setOpenMenu] = useState(false);
+  const [values, setValues] = useState({
+    title: "",
+    text: "",
+    color: "default",
+    archived: false,
+  });
```

Comment:
the state is storing an object, this could cause a performance issue if the state grows and new attributes are added, having a separate state for each input could be a better approach